src/main/resources/lang/zh_cn/messages.yml

```
@@ -563,4 +560,10 @@ BTN_CONFIG_STATUS_UNSUCCESSFUL_HTTP_REQUEST: "向 URL {} 发送 HTTP 请求失
 BTN_CONFIG_STATUS_UNSUCCESSFUL_INCOMPATIBLE_BTN_PROTOCOL_VERSION_CLIENT: "当前客户端 BTN 协议实现版本过旧，本地实现版本号: {}，服务器要求最低: {}"
 BTN_CONFIG_STATUS_UNSUCCESSFUL_INCOMPATIBLE_BTN_PROTOCOL_VERSION_SERVER: "当前服务器 BTN 协议实现版本过旧，服务器实现版本号: {}，本地要求最低: {}"
 BTN_CONFIG_STATUS_SUCCESSFUL: "已成功连接到 BTN 服务器"
-BTN_CONFIG_STATUS_EXCEPTION: "配置 BTN 功能时发生意外错误：{}: {}"
\ No newline at end of file
+BTN_CONFIG_STATUS_EXCEPTION: "配置 BTN 功能时发生意外错误：{}: {}"
+DOWNLOADER_QB_FAILED_REQUEST_TRACKER_LIST_ON_TORRENT: "请求种子 {} 的 Tracker 列表失败：{} - {}"
+TRACKER_FAILED_TO_UPDATE_TRACKER: "请求更新种子 {} 的 Tracker 列表失败：{} - {}"
+UTILITIES_TRACKER_REPLACED: "已成功替换 {} 个种子的 Tracker"
+DOWNLOADER_BIGLYBT_INCORRECT_ADAPTER_VERSION: "当前使用的 BiglyBT 适配器版本不受支持，请升级到 {} 或更高版本"
+LAB_EXPERIMENT_FILL_MISSING_DATA_IN_TRAFFIC_SUMMARY_TITLE: "使用已知数据填充流量统计缺失的数据"
+LAB_EXPERIMENT_FILL_MISSING_DATA_IN_TRAFFIC_SUMMARY_DESCRIPTION: "使用上一条的已知的数据尝试填充流量统计中缺失的数据以尝试修正错误的流量统计数据（仅在非开头区域中缺失记录时有效）。这是一种实验性算法，且提供的数据可能仍旧不准确。"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)